<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OcctQtControls</class>
 <widget class="QWidget" name="OcctQtControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>581</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="10">
      <widget class="QToolButton" name="toolButton_zoom_plus">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/zoom_plus.svg</normaloff>:/zoom_plus.svg</iconset>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>50</number>
       </property>
       <property name="autoRepeatInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QToolButton" name="toolButton_zoom_front">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-front.svg</normaloff>:/view-front.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QToolButton" name="toolButton_zoom_extends">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/zoom_extend.svg</normaloff>:/zoom_extend.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="toolButton_zoom_3d">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-iso.svg</normaloff>:/view-iso.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2000</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="6">
      <widget class="QToolButton" name="toolButton_zoom_left">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-left.svg</normaloff>:/view-left.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="toolButton_zoom_back">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-back.svg</normaloff>:/view-back.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="toolButton_zoom_bottom">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-bottom.svg</normaloff>:/view-bottom.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <widget class="QToolButton" name="toolButton_zoom_min">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/zoom_min.svg</normaloff>:/zoom_min.svg</iconset>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
       <property name="autoRepeatDelay">
        <number>50</number>
       </property>
       <property name="autoRepeatInterval">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QToolButton" name="toolButton_zoom_right">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-right.svg</normaloff>:/view-right.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QToolButton" name="toolButton_zoom_top">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="libicon/icons.qrc">
         <normaloff>:/view-top.svg</normaloff>:/view-top.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>135</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Show perspective</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Show orthographic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Show wireframe</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Show shaded</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Show triedron</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hide triedron</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Show boundary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hide boundary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Show cube</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hide cube</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="4">
      <widget class="QLabel" name="label_mouse_z">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label_val_mouse_z">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_mouse_y">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_val_mouse_y">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_val_mouse_x">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>0.000</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_mouse_x">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2000</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="QSlider" name="verticalSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>2000</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="libicon/icons.qrc"/>
 </resources>
 <connections/>
</ui>
